alchemist/alchemist-engine/src/main/java/it/unibo/alchemist/core/implementations/Engine.java

```diff
@@ -376,6 +380,10 @@ public void run() {
                 L.error("The simulation engine crashed.", e);
             } finally {
                 status = Status.TERMINATED;
+                times[1] = System.nanoTime();
+                //CHECKSTYLE:OFF
+                L.info("Thread " + Thread.currentThread().getId() + " execution time: " + Double.toString((times[1] - times[0]) / 1000000000.0));
```

Comment:
This can't get into the mainline. Moreover, SLF4J uses higher performance calls:
``L.info("Thread  {} execution time: {}", Thread.currentThread().getId(), Double.toString((times[1] - times[0]) / 1000000000.0)``